src/interfaces/assistants_web/package.json

```diff
@@ -9,7 +9,7 @@
     "yarn": "9999"
   },
   "scripts": {
-    "dev": "next dev --port 4444 --turbo",
```

Comment:
Which was the issue?

Reply:
styles where not correct for syntax highlight =/ will resolve later when we get more time